_posts/2015-05-15-Rust-1.0.md

```diff
@@ -0,0 +1,331 @@
+constantly changing.
+
+**The 1.0 release marks the end of that churn.** This release is the
+official beginning of our [commitment to stability][stable], and as
+such it offers a firm foundation for building applications and
+libraries. From this point forward, breaking changes are largely out
+of scope (some [caveats] apply, such as compiler bugs).
+
+That said, releasing 1.0 doesn't mean that the Rust language is
+"done". We have many [improvements in store][priorities]. In fact, the
+Nightly builds of Rust already demonstrates [improvements to][24965]
+[compile][24615] [times][25323] (with more to come) and includes work
+on new APIs and language features, like [`std::fs`][1044] and
+[associated constants][23606].
+
+To help ensure that compiler and language improvements make their way
+out into the ecosystem at large as quickly as possible, we've adopted
+a [train-based][train] release model. This means that we'll be issuing
+regular releases every six weeks, just like evergreen browsers. **To
+kick off that process, we are also releasing Rust 1.1 beta today,
+simultaneously with Rust 1.0.**
+
+### Cargo and crates.io
+
+Building a real project is about more than just writing code -- it's
+also about managing dependencies. [Cargo][cargo], the Rust package
+manager and build system, is designed to make this easy.  Using Cargo,
+downloading and installing new libraries is as simple as adding one
+line to your manifest.
+
+Of course, to use a dependency, you first have to find it. This is
+where [crates.io] comes in -- crates.io is a central package
+repository for Rust code. It makes it easy to search for other
+people's packages or to publish your own.
+
+Since we [announced cargo and crates.io][cargo] approximately six
+months ago, the number of packages has been growing
+steadily. Nonetheless, it's still early days, and there are still lots
+of great packages yet to be written. If you're interested in building
+a library that will take the Rust world by storm, there's no time like
+the present!
+
+### Open Source and Open Governance
+
+Rust has been an open-source project from the start. Over the last few
+years, we've been constantly looking for ways to make our governance
+more open and community driven. Since we introduced the
+[RFC process][rfcs] a little over a year ago, all major decisions
+about Rust are written up and discussed in the open in the form of a
+RFC. Recently, we adopted a [new governance model][1068], which
+establishes a set of subteams, each responsible for RFCs in one
+particular area. If you'd like help shape the future of Rust, we
+encourage you to get involved, either by uploading libraries to
+[crates.io], commenting on RFCs, or
+[writing code for Rust itself][contributing].
+
+We'd like to give a special thank you to the following people, each of
+whom contributed changes since our previous release (the
+[complete list of contributors][AUTHORS] is here):
```

Comment:
Recommend mentioning the total number of contributors to Rust 1.0, perhaps with "each of whom contributed changes since our previous release, bringing the total number of contributors to over a thousand".